lib/svg-sprite.js

```diff
@@ -323,13 +326,14 @@ SVGSpriter.prototype._indexNamespace = function(index) {
  * @param {Function} cb             Callback
  */
 SVGSpriter.prototype._layout = function(config, cb) {
-    const tasks = [];
     const files = {};
     const layout = new Layouter(this, config);
     const createLayoutTask = (k, m) => _cb => {
         layout.layout(files, k, m, _cb);
     };
 
+    const tasks = [];
+
     for (const [mode, value] of Object.entries(config)) {
```

Comment:
Maybe it's better to replace with `.map()` instead?

Reply:
Not sure if `map` would work, maybe we need `reduce`. I say we leave this for another time :)